app/uk/gov/hmrc/debttransformationstub/controllers/DebtManagementAPITestController.scala

```diff
@@ -57,4 +54,37 @@ class DebtManagementAPITestController @Inject() (
         }
   }
 
+  def getDebtDataAndDWISignals(wmfId: String): Action[AnyContent] = Action.async { request =>
+    if (appConfig.isPollingEnv)
+      pollingService.insertRequestAndServeResponse(Json.obj(), request.uri).map {
+        case Some(response) => Status(response.status.getOrElse(200))(response.content)
+        case None => ServiceUnavailable
+      }
+    else
+      environment.getExistingFile(s"$basePath/dm/subcontractor/wmfId.json") match {
+        case None => Future.successful(NotFound("file not found"))
+        case Some(file) =>
+          val result = Source.fromFile(file).mkString.stripMargin
+          Future.successful(Ok(result))
+      }
+  }
+
+  def fieldCollectionsTemplates(): Action[FCTemplateRequest] =
```

Comment:
merged these dmapi requests together in the same controller

Reply:
Nice!